build-packages/magento-scripts/lib/commands/start.js

```diff
@@ -111,9 +113,15 @@ module.exports = (yargs) => {
             logger.logN(`Magento Admin panel credentials: ${logger.style.misc(magentoConfiguration.user)} - ${logger.style.misc(magentoConfiguration.password)}`);
             logger.note(`MySQL credentials, containers status and project information available in ${logger.style.code('npm run status')} command.`);
             logger.log('');
+
+            if (process.isLegacy) {
+                await googleAnalytics.trackTiming('CMA first development time', new Date().getTime() / 1000 - timeStamp);
```

Comment:
Maybe instea of `new Date().getTime()` use `Date.now()` ?

Reply:
Yea, seems that .now is a lot faster